examples/run/dp1.x-lammps-ABACUS-lcao-dpks/methane/machine.json

```diff
@@ -0,0 +1,88 @@
+{
+	"train":[
+	{
+	"machine":{
+		"machine_type": "slurm",
+		"host_name": "localhost",
+		"port": 22,
```

Comment:
Could you please update examples compatible with the new version of `dpdispatcher` ?   Search "new dpdispatcher update note" in the manual https://github.com/deepmodeling/dpgen#set-up-machine

Reply:
Maybe I should remove the machine.json here and the same file in ABACUS-pw? The machine json file has no difference with the vasp example except for the "fp" command.

Reply:
I have updated the file according to the https://github.com/deepmodeling/dpgen#set-up-machine. You could remove the machine.json as well. I think this example file is not that necessary.